lib/new_relic/agent/configuration/default_source.rb

```diff
@@ -1792,7 +1792,10 @@ def self.enforce_fallback(allowed_values: nil, fallback: nil)
           :type         => Array,
           :allowed_from_server => true,
           :transform    => DefaultSource.method(:convert_to_regexp_list),
-          :description  => 'Define transactions you want the agent to ignore, by specifying a list of patterns matching the URI you want to ignore.'
+          :description  => 'Define transactions you want the agent to ignore, by specifying a list of patterns matching the URI you want to ignore.' \
```

Comment:
I believe the comma before the word "by" should be removed. Otherwise, this looks good to me!